internal/apischema/openapi3.json

```diff
@@ -6049,56 +6079,6 @@
         }
       }
     },
-    "/v2/managed_resources": {
```

Comment:
Is this removal intentional?

Reply:
Yes, I've copied over all the schemas which is what lojo was expecting when he made the change. We only read 'secret' but to make it harder to accidentally pull in the wrong schema we now have all of them in a single folder